### ISSUE TITLE
reset-password fails to complete setup

### ISSUE THREAD

**Issue by pczjrh** [symfony/maker-bundle#1294]
Hi,
Running php bin/console make:reset-password
Symfony 5.4, fully up to date, no deprecation notices etc
All doctrine/ packages are latest versions.

I get this error:
```
 created: src/Controller/ResetPasswordController.php
 created: src/Entity/ResetPasswordRequest.php

In DoctrineHelper.php line 91:
                                                                              
  Cannot find the entity manager for class "App\Entity\ResetPasswordRequest"  
                                                                              
make:reset-password
```
The bundle fails to create the remaining files required to complete set-up.
Please advise on any solutions, or what info you need to help?
Cheers,
Jason.

**Comment by weaverryan** [comment]
Hi Jason!

Very strange. Is there anything unique to your Doctrine setup?

**Comment by pczjrh** [comment]
Hi Ryan.
No, not at all. Pretty much exactly out of the symfonycasts tutorials. I've updated all my code to 5.4, and all of the recipies. 
Ran it again after a clean up, and got the same error.
Jason.

**Comment by jrushlow** [comment]
Howdy! This is caused by using something other than `attribute` mapping for the entities. In "newer" Symfony versions, attribute mapping is supported and configured out of the box. When upgrading _to_ newer versions, you'll run into this error if `doctrine.orm.mapping_type` is set to say, `annotation` See: https://symfony.com/doc/current/reference/configuration/doctrine.html#type

The preferred way to handle entity mapping configuration is to remove `doctrine.orm.mapping_type` if it exists and set `doctrine.orm.auto_mapping: true` 

Hope this helps!

fixed in #1475
